.prettierrc

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
This repository currently uses the `standard` linting tool. 

Would it be possible to remove this config file and have folks just use `standard` instead of adding more tooling? 😅

Reply:
oops! was not supposed to commit that  :rofl: